components/DesktopInventoryItemList/index.tsx

```diff
@@ -9,19 +9,9 @@ import TablePagination from '@mui/material/TablePagination'
 import TableRow from '@mui/material/TableRow'
 import TableSortLabel from '@mui/material/TableSortLabel'
 import { visuallyHidden } from '@mui/utils'
-import DesktopInventoryItemListItem from 'components/DesktopInventoryItemList/DesktopInventoryItemListItem'
+import InventoryItemListItem from 'components/DesktopInventoryItemList/DesktopInventoryItemListItem'
 import { InventoryItemResponse } from 'utils/types'
-
-interface Data {
-  name: string
-  attributes: string
-  category: string
-  quantity: number
-  assignee: string
-  kebab: string
-}
-
-const rows = []
+import { Data } from './types'
```

Comment:
```suggestion
import { Data } from 'components/DesktopInventoryItemList/DesktopInventoryItemListItem/types'
```

Use absolute imports, see #35